assets/js/admin/course-pricing-promo-sidebar.js

```diff
@@ -0,0 +1,60 @@
+import { __ } from '@wordpress/i18n';
+import { applyFilters, addFilter } from '@wordpress/hooks';
```

Comment:
I don't think we're using `addFilter` here.

Reply:
Jup, used it for testing, forgot to remove, [removed](https://github.com/Automattic/sensei/pull/4810/files#diff-0cdf94ac1cfb0a1e90adaef288fdb975f07f4c39ff34e0de2003256181b3ec93R2)